README.adoc

```diff
@@ -183,7 +183,7 @@ Properties defined in the first files will take precedence over properties speci
 
 A few special properties do not follow the above mechanism:
 
-* `mvnd.daemonStorage`: this property defines the location where mvnd stores its files (registry and daemon logs).  This property can only be defined as a system property on the command line
+* `mvnd.daemonStorage`: this property defines the location where mvnd stores its files (registry and daemon logs).  This property can only be defined as a system property into the command line
```

Comment:
I think that "on" is more idiomatic in this context :upside_down_face: (Not sure I've ever seen "into" used this way.)

Reply:
Aha good catch I must have overwritten and then modified the wrong "on" when I force pushed.